src/js/migrations.js

```diff
@@ -181,6 +181,17 @@ exports.Migrations= {
     }
   },
 
+  unblockEff: function(badger) {
```

Comment:
Do we actually need the migration?

Reply:
It is for good measure, not all subdomains are DNT compliant. So if `eff.org` is blocked, then `act.eff.org` will still load, but  `anon-stats.eff.org` will be blocked bc it isn't DNT compliant.

Reply:
It seems weird how DNT isn't reversible: toggling DNT compliance checking (and unblocking) off, visiting some DNT compliant domains (that seem to perform tracking) and then toggling DNT compliance checking back on does not in fact get you back to where you would have been had you never fiddled with DNT checking.

Reply:
Let's outline what happens with `eff.org` domains to help understand this situation:
1. Disable DNT checking.
2. Visit `https://stopsesta.org/`, `https://savecrypto.org/`, `https://necessaryandproportionate.org/sign`.
3. `act.eff.org` is seen perform tracking on three sites; it and its parent (`eff.org`) are marked for blocking.
4. Enable DNT checking.
5. Visit the sites again.
6. `anon-stats.eff.org` is still blocked (even though it was never seen tracking), because `eff.org` is still marked for blocking in `action_map`.

When DNT checking is enabled, `act.eff.org` would not (ordinarily) get marked for blocking more than once and so the `eff.org` entry would be there, but set to "allow" instead of "block".

Reply:
I don't think we need to fix this now, but it seems like a bug. If we handled DNT better, we wouldn't need a migration, for example.

Reply:
Would you mind making an issue for handling DNT better?

Reply:
To be clear, the above steps are with this PR's patch applied. Without the patch, `act.eff.org` would not get rechecked for DNT compliance in step 5.